src/pytest/test_spell_check.py

```diff
@@ -0,0 +1,158 @@
+from rmtest import BaseModuleTestCase
+import redis
+import unittest
+from hotels import hotels
+import random
+import time
+
+
+class SpellCheckTestCase(BaseModuleTestCase):
+
+    def testDictAdd(self):
+            self.cmd('flushdb')
```

Comment:
Don't need to do this. Will be an error in the new framework.

Reply:
Will be fixed when I merge master into this branch